projects/plugins/jetpack/modules/concat/class.jetpack-scripts.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+class JetPack_Scripts extends WP_Scripts {
+
+	/**
+	 * Constructor.
+	 *
+	 * @since 2.6.0
```

Comment:
For all those `@since` values, you can use the `$$next-version$$` placeholder. It will get replaced by the proper version during the release.